client/elements/menus/language-menu.html

```diff
@@ -188,6 +189,10 @@
               }
           }
 
+          _shouldShowMenu(languageChoice) {
+            return languageChoice.filter(v => {return v.lang !== this.inputLanguage}).length > 0;
```

Comment:
1. change `filter()` to `some()`.
2. Change `v` to a more descriptive name
3. Use the shorthand form without `{}` and `return`